app/templates/rule_scheduled_change_modal.html

```diff
@@ -0,0 +1,221 @@
+<div class="modal-header">
+    <h3 class="modal-title" ng-show="is_edit">Edit Scheduled Change</h3>
+    <h3 class="modal-title" ng-show="!is_edit">Add Scheduled Change</h3>
+</div>
+<div class="modal-body">
+  <form role="form">
+    <h4>Scheduled Change Details</h4>
+    <div class="row text-center">
+      <button id="btn_time" class="btn active" type="button" ng-click="toggleType('time')">Time Based</button>
+      <button id="btn_telemetry" class="btn" type="button" ng-click="toggleType('telemetry')">Telemetry Based</button>
```

Comment:
I really like that having an either/or for When/Telemetry. Looks like we can add 'disabled' to class here so people don't try to use it yet.